components/ENSLookup.tsx

```diff
@@ -1,11 +1,7 @@
 import useENSName from "../hooks/useENSName";
 
-type Address = {
-  address: string | string[];
-};
+import type { Address } from "./types";
 
-const ENSLookup = ({ address }: Address) => {
+export const ENSLookup = ({ address }: Address) => {
```

Comment:
I like named exports over default most of the time because default exports aren't locked to a name and can be called whatever someone pleases... which, can get confusing. IMO.